cars/v1/x_pack/README.md

```diff
@@ -1,18 +1,29 @@
-This directory contains example configurations for x-pack:
+# This directory contains example configurations for x-pack:
 
 * `security`: Configures TLS for all HTTP and transport communication using self-signed certificates.
-* `monitoring-local`: Enables x-pack monitoring with export to the current cluster.
+* `monitoring`: Enables x-pack monitoring with export to the current cluster.
 
 The configurations have been implemented so that you can either only one of them or both together, i.e. all of the following combinations will work:
 
-* `--elasticsearch-plugins="x-pack:security"`
-* `--elasticsearch-plugins="x-pack:monitoring-local"`
-* `--elasticsearch-plugins="x-pack:security+monitoring-local"`
+* `--car-plugins="x-pack-security"`
+* `--car-plugins="x-pack-monitoring-local"`
+* `--car-plugins="x-pack-security,x-pack-monitoring-local"`
 
-The `security` configuration will enable basic authentication and TLS for the HTTP and the transport layer. It will also add a `rally` super-user with the password `rally-password`. As a consequence, you will need to add the following client options when benchmarking this configuration:
+## Configuring security user name, password, role
```

Comment:
In `repository-gcs` we've called this section `Parameters`. Maybe we should align it?